src/main/main.ts

```diff
@@ -190,10 +190,13 @@ app.on('ready', () => {
 });
 
 app.on("web-contents-created", (_event: any, webContents: WebContents) => {
-    // Prevent navigation
+    // Prevent navigation to local links
```

Comment:
@krassowski do you have any concerns on these changes? any concerns navigating to external websites?

Reply:
Yes, I am concerned. Wouldn't this lead to a context leak? Why is this change needed, again?

Reply:
during my testing, external domain urls were opened in popup windows and that is safe, so I thought we could enable it back. but if you think there are cases that require blocking external urls then I can update it to block. The original motivation for this change was the issue I mentioned in earlier comments: `After switching to Classic notebooks and clicking on a notebook opens a blank window`